### PR TITLE
Fix 198 prop type and docs

### PR DESCRIPTION
**Pull request by prometheas**
Addresses issue #198.

### CODE REVIEW

**Comment by prometheas** [comment]
Hey gang. I've opened up a support ticket, too, but I'd really like to get rid of that incorrect proptype warning at the bottom of my simulator. It's not a good look for our demos 😉

File: README.md [hunk 1]
```diff
@@ -152,22 +153,22 @@ class PlaidEventContainer extends React.Component {
...
-      <PlaidLink
-        token = {<#GENERATED_LINK_TOKEN#>}
-        component= {Button}
-        componentProps = {{title: 'Add Account'}}
-        onSuccess = {(result) => {console.log('Success: ', result)}}
-        onError = {(result) => {console.log('Error: ', result)}}
-    >
+```jsx
+<PlaidLink
+  token = {/* <#GENERATED_LINK_TOKEN#> */}
```
Comment (amytang0) [inline README.md:168]:
```suggestion
  token = {"<#GENERATED_LINK_TOKEN#>"}
```

**Review by amytang0 — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -91,17 +92,17 @@ project(':react-native-plaid-link-sdk').projectDir = new File(rootProject.projec
...
   return (
     <PlaidLink
-     // Replace any of the following <#VARIABLE#>s according to your setup,
-     // for details see https://plaid.com/docs/quickstart/#client-side-link-configuration
- 
-      token={<#GENERATED_LINK_TOKEN#>}
+      // Replace any of the following <#VARIABLE#>s according to your setup,
+      // for details see https://plaid.com/docs/quickstart/#client-side-link-configuration
+
+      token={/* <#GENERATED_LINK_TOKEN#> */}
```
Comment (amytang0) [inline README.md:105]:
```suggestion
      token = {"<#GENERATED_LINK_TOKEN#>"}
```

**Review by amytang0 — APPROVED** [review]
Hey Prometheas, thank you so much for putting out this PR! 
I changed some of the README.md around the token field, but otherwise LGTM

**Comment by prometheas** [comment]
image

**Comment by prometheas** [comment]
Any guidance as to when this fix will go out to npm?

**Pull request merged**